auth-server/src/services/email.ts

```diff
@@ -49,17 +48,14 @@ export const sendVerificationEmail = (user: User, token: EmailVerificationToken)
 		console.error('Verification Email not sent', e));
 };
 
-export const sendResetPasswordEmail = (user: User, token: PasswordResetToken): void => {
+export const sendResetPasswordEmail = (user: User, token: string): void => {
 	if (!apiKey) {
 		console.warn('Password reset Email not sent due to missing API key');
 		return;
 	}
 
-	const resetUrl = `${DOMAIN}/reset-password/${token.token}`;
-	const text = ejs.render(
-		resetPasswordEmailTemplate,
-		{ resetUrl, username: user.name || '' }
-	);
+	const resetUrl = `${DOMAIN}/reset-password/${token}?user_id=${user.id}`;
```

Comment:
let's use query string for both, `?token=abc&userId=123`

Reply:
done